backend/app/core/conf.py

```diff
@@ -75,6 +84,9 @@ class Settings(BaseSettings):
     MIDDLEWARE_GZIP: bool = True
     MIDDLEWARE_ACCESS: bool = False
 
+    class Config:
+        env_file = '.env'
+
 
 @lru_cache
```

Comment:
Here's an async cacher that I'm not sure can be replaced: [async-lru](https://github.com/aio-libs/async-lru)

Reply:
Please check #16